audio-worklet/src/bitcrusher-demo.js

```diff
@@ -85,16 +113,50 @@ class BitcrusherDemo {
           default: this.masterGain_.gain.value,
           name: 'Volume'
         });
+
+    // GUI controls allow switching between audio worklet and script processor.
+    // The audio worklet node is used by default.
+    this.useAudioWorkletCheckBox_ = document.createElement('input');
+    this.useAudioWorkletCheckBox_.type = 'checkbox';
```

Comment:
Why not a radio button?  I'm assuming the demo must either use the script OR the worklet, not both, so two buttons would seem more natural to select which mode you want to try.